### PR TITLE
Added: Filter by Date, Priority, Labels

### PR DESCRIPTION
**Pull request by saurabhchirde**
Added Filter features and Label Page

Using demoNotes for testing and review purposes only. 
If you add any new notes, all demo notes will get vanished.

Filter By:
1. Date
2. Priority
3. Label

label-filter

### CODE REVIEW

File: src/Components/UI/NoteLabel/NoteLabel.jsx [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
 
 const NoteLabel = () => {
   const { label, setLabel, dispatch } = usePebbleNote();
+  const { alertDispatch } = useAlert();
 
   const submitLabel = (e) => {
     e.preventDefault();
-    dispatch({ type: "addLabelToNote", payload: label });
+    if (label.trim() === "") {
+      alertDispatch({ type: "emptyLael" });
```
Comment (SinchanaMS) [inline src/Components/UI/NoteLabel/NoteLabel.jsx]: Please check the type. Spelt wrong.

**Review by SinchanaMS — APPROVED** [review]
The code is very neat and easy to understand. Good work, Saurabh!

File: src/Utils/dateFormatter.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const dateFormatter = (a) => {
+  const date = Number(
+    `${a.date.slice(6)}${a.date.slice(3, 5)}${a.date.slice(0, 2)}`
+  );
+  console.log(date);
```
Comment (deekshithmd) [inline src/Utils/dateFormatter.js:5]: suggestion: You can remove console.log if not required anymore

**Review by deekshithmd — APPROVED** [review]

File: src/Components/UI/NoteLabel/NoteLabel.jsx (hunk 1 above)
Comment (saurabhchirde) [reply src/Components/UI/NoteLabel/NoteLabel.jsx]: Thank you for pointing it out. Will Fix it.

File: src/Utils/dateFormatter.js (hunk 2 above)
Comment (saurabhchirde) [reply src/Utils/dateFormatter.js:5]: Oh yes, there is no use, I will remove it.

**Review by keyurshah339 — COMMENTED** [review]
good to merge

**Pull request merged**